router/index.js

```diff
@@ -16,13 +16,13 @@ router.get("/logout", route.logout)
 router.post("/register", auth.doRegister)
 router.post("/login", auth.doLogin)
 
-// profiles
+// dynamic profiles

```

Comment:
I like the addition of the word dynamic 😃 
I don't think we need comments here, but it all seems fine as it is.  
Again, if Deanna agrees with structuring the controller object the way you did we can go ahead and merge & create a new issue for the handlebarsHelper-file.

Reply:
Tbh I'm (almost) always pro-comments as it rarely pollutes the file and helps other devs understanding the file more quickly.
I say: we keep it!